### PR TITLE
added proteogenomics to the GTN

### PR DESCRIPTION
**Pull request by subinamehta**
Hi,

I have created tutorial for the proteogenomics workflow! it is in 3 parts, db creation, db searching and novel peptide analysis. Let me know if I need to add more information

Subina

### CODE REVIEW

**Comment by subinamehta** [comment]
I have updated the Readme.md with proteogenomics too!

File: _config.yml [hunk 1]
```diff
@@ -131,3 +131,5 @@ exclude:
 plugins:
   - jekyll-feed
   - jemoji
+
+theme: jekyll-theme-cayman
```
Comment (bgruening) [inline _config.yml]: Can this change be removed?

File: topics/proteomics/tutorials/Proteogenomics/Proteogenomics_DBsearch/Protegenomics_RNAseqDBworkflow.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
+install_tool_dependencies: True
```
Comment (bgruening) [inline topics/proteomics/tutorials/Proteogenomics/Proteogenomics_DBsearch/Protegenomics_RNAseqDBworkflow.yaml]: can we have the folder name `Proteogenomics` be with small letters, like `proteogenomics`.

File: _config.yml (hunk 1 above)
Comment (subinamehta) [reply _config.yml]: Yup! just removed it!

File: topics/proteomics/tutorials/Proteogenomics/Proteogenomics_DBsearch/Protegenomics_RNAseqDBworkflow.yaml (hunk 2 above)
Comment (subinamehta) [reply topics/proteomics/tutorials/Proteogenomics/Proteogenomics_DBsearch/Protegenomics_RNAseqDBworkflow.yaml]: I dont know how to edit it and write it in small letters. Whenever I do something like that, it creates a new folder. Do you have any suggestions on how to rename/edit it??

Comment (bgruening) [reply topics/proteomics/tutorials/Proteogenomics/Proteogenomics_DBsearch/Protegenomics_RNAseqDBworkflow.yaml]: `git mv topics/proteomics/tutorials/Proteogenomics topics/proteomics/tutorials/proteogenomics` should work afaik and then commit the change

**Comment by subinamehta** [comment]
Made a few more changes to the files!

**Comment by bebatut** [comment]
@subinamehta thanks for your contribution

Could you move the tutorial folders from `topics/proteomics/tutorials/proteogenomics` to `topics/proteomics/tutorials/tutorials`? And then reference them in the `topics/proteomics/metadata.yaml` file?
Using this structure (detailed [there](http://galaxyproject.github.io/training-material/topics/contributing/tutorials/create-new-tutorial/tutorial.html)), the tutorials will be listed in http://training.galaxyproject.org/topics/proteomics/

To make sure that the `proteogenomics` information is visible in the proteomics page, you can add a tag for these tutorials in the `metadata.yaml` file, like:

```
   tags:
      - proteogenomics
```

**Comment by subinamehta** [comment]
I have moved the files and changed the metadata.yaml file!!

**Comment by subinamehta** [comment]
Should I send a new pull request??

**Comment by bebatut** [comment]
Thanks @subinamehta.
I will check this PR today (no need to open a new one)

**Comment by shiltemann** [comment]
Hi @subinamehta we have changed the way we organize metadata in this repository, and I have updated your PR accordingly. You will now find the metadata at the top of the `tutorial.md` files

I also noticed you have 3 tutorial, but only had one metadata entry, so I have copied this to all 3 tutorials, but please change the metadata for each as necessary :)

**Comment by subinamehta** [comment]
I have made the required changes...please check them!! Thanks! appreciate it!

**Comment by bgruening** [comment]
@subinamehta thanks a lot for working on this!

File: CONTRIBUTORS.yaml [hunk 3]
```diff
@@ -216,6 +216,8 @@ joachimwolff:
 
 jpmaree:
 
+jraysajulga:
```
Comment (shiltemann) [inline CONTRIBUTORS.yaml]: @jraysajulga Thanks for contributing to this training material! We added you to our [Hall of Fame](https://galaxyproject.github.io/training-material/hall-of-fame), please add any information here you want (name, contact details)

ping @subinamehta

Comment (shiltemann) [reply CONTRIBUTORS.yaml]: thanks!

**Pull request merged**